src/main/resources/assets/blocky-bubbles/lang/ru_ru.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Translation for summary and description in ModMenu (summary from Modrinth, description from fabric.mod.json)

```suggestion
{
  "modmenu.summaryTranslation.blocky-bubbles": "Порт «Быстрых пузырей» из Bedrock Edition.",
  "modmenu.descriptionTranslation.blocky-bubbles": "Позволяет отображать колонны пузырей в виде блока, что внешне напоминает «Быстрые пузыри» из Bedrock Edition.",
```

Reply:
Also @axialeaa , you may missed that — what's your final opinion on this?
![image](https://github.com/user-attachments/assets/aad02668-3e4b-41f5-b90b-51f3006060f1)
Should we solve this in localization or wait until the source string is adjusted?